python-dict-attribute/config_v2.py

```diff
@@ -0,0 +1,21 @@
+class Config:
```

Comment:
This file doesn't seem to match the one you discuss [in the tutorial](https://realpython.com/python-dict-attribute/#deleting-attributes).

I haven't updated this, because I'm not sure what you intended to use. Please double-check and update this.